zebra/redistribute.c

```diff
@@ -472,6 +474,9 @@ void zebra_interface_down_update(struct interface *ifp)
 
 		zsend_interface_update(ZEBRA_INTERFACE_DOWN, client, ifp);
 	}
+
+	RB_FOREACH (n, zebra_neigh_rb_head, &zneigh_info->neigh_rb_tree)
```

Comment:
seems like we should call a function implemented in zebra_neigh.c that hides the details of the rb head from redistribute.c to clean up

Reply:
You mean we should create a new function in zebra_neigh.c that actually does RB_FOREACH stuff and call that function here?

Reply:
Fixed.